Computer_Architecture/CA_3.11_Floating-Point_Code.md

```
@@ -0,0 +1,39 @@
+- 세월이 흐르며 MMX → SSE(Streaming SIMD extensions) → AVX(Advanced vector extensions) 로 변경되었다.
+- SSE 의 경우 XMM 이라 불리는 레지스터 집합에서 데이터를 관리한다. XMM은 128비트이다.
+- AVX는 YMM이라 불리는 레지스터 집합에서 데이터를 관리하는데, YMM은 256 비트로, 각 YMM 레지스터는 8개의 32비트 값, 4개의 64비트 값을 저장할 수 있고, 정수 혹은 부동 소수점일 수 있다.
+
+- 2000년 Pentium 4와 도입된 SSE2부터 미디어 명령어는 XMM 또는 YMM 레지스터의 최하위 32비트 또는 64비트에 단일 값을 사용하여 스칼라 부동 소수점 데이터를 처리한다.
+- 모든 x86-64 코드를 실행할 수 있는 프로세서는 SSE2 또는 그 이상을 지원하며, 따라서 x86-64 부동 소수점은 SSE 또는 AVX에 기반한다.
+
+- 해당 도서는 Core i7 Haswell 프로세서와 함께 2013년에 도입된 SIMD(단일 명령 다중 데이터) 확장인 AVX2를 기반으로 한다.
+
+## Media Registers
+
+![](https://i.imgur.com/2XU3dyP.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)